speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SQLDatabaseStorage.cpp

```diff
@@ -337,15 +333,29 @@ int SQLDatabaseStorage::InsertInitialData(std::ifstream& p_inputFile)
     GET_INT_FROM_RESULTS(environmentId)
 
     // settings
+    // Saved as enum index, but since indices in our code and MySQL are not the same, perform conversion
     std::string interventionMode;
-
     READ_INPUT(p_inputFile, interventionMode);
-
-    values = "'" + interventionMode + "'";
+    switch (std::stoi(interventionMode)) {
+        case INTERVENTION_TYPE_NO_SIGNALS:
+            values = "'Off'";
+            break;
+        case INTERVENTION_TYPE_ONLY_SIGNALS:
+            values = "'Suggest'";
+            break;
+        case INTERVENTION_TYPE_SHARED_CONTROL:
+            values = "'Shared'";
+            break;
+        case INTERVENTION_TYPE_COMPLETE_TAKEOVER:
+            values = "'Force'";
+            break;
+        default:
+            throw std::exception("Invalid intervention type index read from buffer file");
+    }
```

Comment:
You only changed 2 of the 6 test input files. (or maybe you didnt push them?)

Reply:
All test files that are present on the main branch have been updated.